guides/doc-Managing_Hosts/topics/proc_registering-a-host-to-project-using-the-global-registration-template.adoc

```diff
@@ -0,0 +1,174 @@
+[id="registering-a-host-to-project-using-the-global-registration-template_{context}"]
+= Registering a Host to {ProjectName} Using the Global Registration Template
+
+Use the following procedure to register a host to {ProjectNameX}.
+
+With this registration method you can register hosts to {Project} by generating a `curl` command on {Project} and running this command on hosts.
+This method uses the global registration template that gives you complete control over the process of the host registration.
+
+ifeval::["{build}" != "satellite"]
+Note that you can extend the parameters by plug-ins. For more information, see https://github.com/theforeman/foreman/blob/develop/developer_docs/how_to_create_a_plugin.asciidoc.
+endif::[]
+
+.Prerequisites
+* The {Project} user that generates the `curl` command must have the `create_hosts` permission.
+* You must have root privileges on the host that you want to register.
+
+.Procedure
+. In {Project} web UI, navigate to *Hosts* > *Provisioning Templates* and locate and click *Linux registration default*.
+
+. Click the *Association* tab.
+
+. Select the operating systems that you want to register.
+
+. Navigate to *Hosts* > *Operating Systems* and click the operating system that you want to register.
+
+. Click *Submit*.
+
+. Click the *Templates* tab.
+
+. From the *Registration template* list, select *Linux registration default*.
+
+. Click *Submit*.
+
+. Navigate to *Hosts* > *All Hosts* > *Register Host*.
+
+. Optional: If you want to register hosts to {SmartProxy}, enter the following command on the {SmartProxy} to enable required modules:
```

Comment:
did I get it right that users need to run this command on Capsule Server?

Reply:
Yes, but I believe that registration module should be enabled by default.
So first step should be to check if capsule has enabled _Registration_ module (_Infrastructure > Capsules > refresh proxy_ & check if Registration feature is present), and if not, then enable it with the commands.

Reply:
Perfect, thank you